.gitignore

```diff
@@ -22,6 +22,9 @@ bld/
 [Oo]bj/
```

Comment:
https://www.google.co.kr/search?q=Oo+bj%2F&oq=Oo+bj%2F&aqs=chrome..69i57j0l2.8442j0j4&sourceid=chrome&ie=UTF-8

'No useful results'
What's this?